### PR TITLE
Use io.open to read file and update envlist for tox #14

### LINKED ISSUE #14: Running `tox -e 2.7` on mitype gives error
### Subject of the issue
Following the contributors guide, running `tox -e 2.7` on mitype gives error.
```
line 5, in <module>
        with open("README.md", encoding="utf-8") as f:
    TypeError: 'encoding' is an invalid keyword argument for this function
```

### Expected behaviour
Should work fine.

### Actual behaviour
Command errors out with exit status 1

### Your proposed fix (if any)
We gotta modify that line under setup.py. But please be careful, because this file decides how mitype appears on [pypi](https://pypi.org/project/mitype/).

### PR DESCRIPTION
**Pull request by andylim0221**
This fixes #14

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py27,py36,py37
+envlist = {py27,py36,py37}-{2.7,3.6,3.7}
```
Comment (Mithil467) [inline tox.ini:2]: Why this change? Can you please elaborate?

Comment (andylim0221) [reply tox.ini:2]: In my environment, if i run your command, `tox -e 2.7`, it shows undefined environment, i chose to alias the environment list, so that i can run the tox test, and align with your command written in the issue.

Comment (Mithil467) [reply tox.ini:2]: You need to run `python -m tox -e py27` instead :) 
You may also run `python -m tox .`

May you instead replace it with this?
`envlist = py2,py3`

Comment (andylim0221) [reply tox.ini:2]: sure, i will replace with `envlist = py2,py3`

**Comment by andylim0221** [comment]
Now it should be fine. Please take a look.

**Comment by Mithil467** [comment]
Awesome @andylim0221!

Thanks for your contribution :)
